src/pysoundcomparisons/__main__.py

```diff
@@ -197,6 +199,101 @@ def upload_images(args):
                 args.log.info('{0} -> {1} object {2.id}'.format(
                     fname, 'new' if created else 'existing', obj))
 
+@command()
+def rename_soundfile(args):
+    """
+    This command downloads the passed old sound files to a temporary folder, renames the old ones by 
+    simultaneously changing their meta data by using:
+      ffmpeg -i old.ext -metadata key=value -codec copy new.ext
+    deletes the old bitstreams and uploads the new ones with same OID.
+    ffmpeg can be installed via https://www.ffmpeg.org and must be found in a shell call.
+    """
+
+    if len(args.args) != 2:
+        args.log.error("need two arguments: old_file_name new_file_name")
+        return
+
+    # OS independent device for suppressing messages while running a shell command
+    nulldev = open(os.devnull, 'w')
+
+    ffmpeg_cmd = 'ffmpeg'
+    if platform.system() == 'Windows':
+        ffmpeg_cmd = 'ffmpeg.exe'
+
+    # check if ffmpeg can be called on the local machine
+    if call("%s -version" % (ffmpeg_cmd), stdout=nulldev, stderr=nulldev, shell=True) != 0:
```

Comment:
That's exactly what [check_call](https://docs.python.org/3/library/subprocess.html#subprocess.check_call) is for. And I wouldn't even bother catching the exception. Isn't it more informative to let the ffmpeg error message just be printed on stderr?